docker-compose.yaml

```diff
@@ -11,6 +11,8 @@ services:
       - SECRET_DATA_PASSWORD=password
       - SECRET_PLATFORM_API_AUTH_SECRET=myAnchorToPlatformSecret
       - SECRET_CALLBACK_API_AUTH_SECRET=myPlatformToAnchorSecret
+      - SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET=secret_sep24_interactive_url_jwt_secret
+      - SECRET_SEP24_MORE_INFO_URL_JWT_SECRET=secret_sep24_more_info_url_jwt_secret
```

Comment:
@lijamie98 I added this two env-var to run the project locally. 
You said it was already defined inside `docker-compose/anchor-get-started/sep-24/docker-compose.yaml`. But I was unsure if I should import all that configuration, so I added only these two. 

If it is not right I can remove them.

Reply:
This file is to be deprecated and deleted. We should have all docker-compose files saved under `docker-compose` folder.